install.sh

```diff
@@ -577,7 +599,8 @@ download_rancher_file() {
           if [ "$(in_no_proxy "${url}")" = "0" ]; then
               noproxy="--noproxy '*'"
           fi
-          RESPONSE=$(curl $noproxy --connect-timeout 60 --max-time 300 --write-out "%{http_code}\n" ${CURL_BIN_CAFLAG} ${CURL_LOG} -fL "${url}" -o "${CATTLE_AGENT_BIN_PREFIX}/bin/${name}")
+          RESPONSE=$($TRANSACTIONAL_CMD curl $noproxy --connect-timeout 60 --max-time 300 --write-out "%{http_code}\n" ${CURL_BIN_CAFLAG} ${CURL_LOG} -fL "${url}" -o "${CATTLE_AGENT_BIN_PREFIX}/bin/${name}")
```

Comment:
I understand the `transactional-update` is supposed to allow for atomic application of changes to a node; however, this command tends to be very chatty in certain cases and can be run multiple times (if you look at the logic below). Is there a way we can mutate this logic to persist the resultant file with transactional-update without the "noise" of running transactional-update N times?

Possible solution for this would maybe be to stage the binary we are downloading to a temporary directory, then proceed to copy or move that binary from the temporary directory into the final destination (provided the transactional update can capture a `cp` or `mv`).

Reply:
Yep, absolutely... with the default `transactional-update -d --no-selfupdate --continue --quiet run` command there's no chat to stdout - the only output is the desired "Successfully downloaded the...", and we can check the response code properly. We chain the changes together, i.e. downloading of the two files, into two operations that are stacked on-top of each other, that's why we use the `--continue` option, as otherwise multiple `transactional-update` commands would drop the previous run's changes. Here there's no need to stage the file and copy it in.